server/core/src/main/java/com/mongodb/stitch/server/core/internal/StitchAppClientImpl.java

```diff
@@ -181,8 +182,17 @@ public void callFunction(
   }
 
   @Override
-  public boolean isLoggedIn() {
-    return getAuth().isLoggedIn();
+  public boolean isLoggedIn() throws InterruptedException {
```

Comment:
[q] If a user ever calls `isLoggedIn()` method on a `StitchAppClient`, is there a case where that can throw? should we mark the method as `throws` in the interface and explain when it could happen?

Reply:
No, all of this is internal.

Reply:
ah you're right, there's no isLoggedIn on the app client itself